components/textfield/index.css

```diff
@@ -131,207 +460,233 @@ governing permissions and limitations under the License.
   */
   -moz-appearance: textfield;
 
-  &::placeholder {
-    font-weight: var(--spectrum-textfield-texticon-placeholder-text-font-weight);
-    font-style: var(--spectrum-textfield-texticon-placeholder-text-font-style);
-    transition: color var(--spectrum-global-animation-duration-100) ease-in-out;
-    opacity: 1;
+  /* Remove the native clear button in IE */
```

Comment:
Does Spectrum CSS still support IE?

Reply:
@pfulton do you know the answer to this question?

Reply:
We do not support IE any longer.